shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -341,9 +346,21 @@ public InputConnection createInputConnection(
       EditorInfoCompat.setContentMimeTypes(outAttrs, imgTypeString);
     }
 
+    EditorInfoCompat.setStylusHandwritingEnabled(outAttrs, true);
+    // TODO(justinmc): Scribe stylus gestures should be supported here via
+    // outAttrs.setSupportedHandwritingGestures and
+    // outAttrs.setSupportedHandwritingGesturePreviews.
+    // https://github.com/flutter/flutter/issues/156018
+
     InputConnectionAdaptor connection =
         new InputConnectionAdaptor(
-            view, inputTarget.id, textInputChannel, keyboardManager, mEditable, outAttrs);
+            view,
+            inputTarget.id,
+            textInputChannel,
+            scribeChannel,
```

Comment:
Instead of adding this parameter, I could make Scribe's methods part of the TextInputChannel. Then I also wouldn't need to pass ScribeChannel into TextInputPlugin in FlutterView.

However, TextInputChannel is pretty bloated, and it's pretty nice to encapsulate all of the Scribe stuff separately, hence why I did  this.